core/fields/Field_Rich.php

```diff
@@ -161,6 +161,55 @@ public function display() {
 					document.addEventListener('click',function(e){
 						// click event handler for editor 
 
+						
+
+						if (e.target.nodeName==='A') {
+							
+							// only work on anchors inside editor (i.e. not toolbar or elswhere... :) )
+							let in_editor = e.target.closest('.editor');
+							if (in_editor) {
+								// remember dynamic editor/textarea
+								window.this_editor = e.target.closest('.control').querySelector('.editor');
+								window.this_textarea = e.target.closest('.control').querySelector('textarea');
+
+								//console.log('show anchor editor');
+								window.editor_anchor = e.target;
+
+								let href = e.target.getAttribute('href');
+								let text = e.target.innerText;
+								let classes = e.target.classList.value;
+								let target = e.target.getAttribute('target') ? e.target.getAttribute('target') : "";
+
+								/* console.log('URL',href);
+								console.log('TEXT',text);
+								console.log('CLASS',classes); */
+								
+								let iL = ["URL", "Display Text", "Class", "Target"];
+								let iI = ["a_url", "a_text", "a_class","a_target"];
+								let cV = [href, text, classes, target];
+								let hL = ["", "", "","'_blank' for new window/tab, otherwise leave empty"];
```

Comment:
please add a comment here as to what this does (probably in php) since these variable names arent the best